pkg/tuned/cmd/operand/cmd.go

```diff
@@ -56,6 +58,7 @@ func NewTunedCommand() *cobra.Command {
 
 func (t *tunedOpts) AddFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&t.inCluster, "in-cluster", true, "In-cluster operand run.")
+	fs.BoolVar(&t.oneShot, "one-shot", false, "Run TuneD in one-shot mode.")
```

Comment:
non-blocking: should `oneShot` imply (and override) `inCluster`?
IOW, should we document and enforce that if `oneShot=true` then `inCluster=false`? This will allow to simplify our logic below.

Reply:
I guess, we could do that.  Not something I'm too worried about though.